cv32e20/tests/programs/custom/hello-world/hello-world.c

```diff
@@ -28,14 +28,14 @@
 #include <stdlib.h>
 
 // MVENDORID CSR: 0x602 is the value assigned by JEDEC to the OpenHW Group
-#define EXP_MVENDORID 0x00000000
-//#define EXP_MVENDORID 0x00000602
+//#define EXP_MVENDORID 0x00000000
```

Comment:
In a future PR it would be good to remove these commented-out EXP_ values as it might create confusion.  It is not worth gating this PR.

Reply:
I will revise this PR and remove the commented out values.